02_Methodiek.Rmd

```diff
@@ -59,45 +59,63 @@ Het kan dan zijn dat het bekomen aantal lichtjes hoger uitvalt.
+
 We willen er ook rekening mee houden dat een beschikbaar meetpunt niet noodzakelijk ook een geschikt meetpunt is.
-Om een geschikt meetpunt te zijn, moet het meetpunt aan kwalitatieve standaarden voldoen.
+
+
 Afhankelijk van die verhoudingen worden de _rastercellen_ **per GT-groep** eerst in _categorieën_ ingedeeld.
 
 #### Indeling van rastercellen in categorieën {#indeling-cat}
 
-_A. Alle meetpunten zijn kwalitatief gelijkwaardig_
+_A. Alle meetpunten met een peilbuis zijn kwalitatief gelijkwaardig_
 
-We stellen eerst voor de eenvoud dat alle beschikbare meetpunten even geschikt zijn: er wordt bvb. geen onderscheid gemaakt tussen een lang en kort bemeten meetpunt. 
-Vergelijken we dan de vraag (het gewenst aantal meetpunten per rastercel) met het aanbod van meetpunten dan kunnen de rastercellen in drie categorieën verdeeld worden:
+We stellen eerst voor de eenvoud dat alle beschikbare meetpunten even geschikt zijn: er wordt bvb. geen onderscheid gemaakt tussen een lang en een kort bemeten meetpunt. 
+Vergelijken we dan de vraag, het gewenst aantal meetpunten per rastercel, met het aanbod van meetpunten met een peilbuis dan kunnen de rastercellen in drie categorieën verdeeld worden:
 
 - Cat. 1: de vraag is groter dan het aanbod: 
+er zijn geen meetpunten met een peilbuis of er is een gebrek aan
 - Cat. 2: de vraag is gelijk aan het aanbod: 
+elk meetpunt met een peilbuis wordt geselecteerd
+- Cat. 3: de vraag is kleiner dan het aanbod: 
+er is een overschot aan meetpunten met een peilbuis 
 elk meetpunt wordt geselecteerd
-- Cat. 3: de vraag is kleiner dan het aanbod: er is een overschot aan meetpunten
 
 **Categorie 1**
-Er is een gebrek aan meetpunten.
-Alle beschikbare meetpunten worden zonder verder analyse geselecteerd.
-_**Als er voor een rastercel een gebrek is aan geschikte meetpunten, stellen we voor om die cel te voorzien van een (nieuw) meetpunt gelegen op een plaats met die GT-groep. **_
-Het gaat hier vooral over een tijdsinvestering: dergelijke meetpunten kunnen pas na enkele jaren effectief meedraaien in het meetnet. 
+Vanuit het oogpunt van een meetnetontwerp kan een rastercel van categorie 1 over twee soorten meetpunten beschikken. 
+Je hebt meetpunten zonder peilbuis en mogelijk ook meetpunten met een peilbuis. 
+Alle meetpunten met een peilbuis worden zonder verder analyse geselecteerd.
+Daarnaast zijn er (vele) kandidaat-meetpunten om te voorzien van een nieuwe peilbuis. 
+
+_**Bovenstaande is een voorstel. Je zou ook kunnen beslissen dat enkel meetpunten met een peilbuis in aanmerking komen voor het meetnet.**_
+_**Of het tegenovergestelde is ook een alternatief: dat voor de ruimtelijke balancering we geen voorkeur geven aan meetpunten met een peilbuis.**_
+_**We vinden dat het huidige voorstel een goede balans vertoont tussen beide.**_
+
+Het gaat hier vooral over een tijdsinvestering: dergelijke meetpunten zullen pas na enkele jaren effectief meedraaien in het meetnet. 
```

Comment:
dit slaat specifiek op de nieuw te installeren peilbuizen, maar die connectie is niet (meer) duidelijk nu